SDK-1204 Updated docs (#686)
box/box-ios-sdk: docs/usage/webhooks.md

----------------

To retrieve information about webhooks in an enterprise, call

> [`client.webhook.list(webhookId: String, marker: String?, limit: Int?, fields: [String]?)`].  This method will return an iterator object in the completion you can use to get the webhooks.

Reviewer: ```suggestion
[`client.webhook.list(webhookId: String, marker: String?, limit: Int?, fields: [String]?)`].  This method will return an iterator object in the completion, which is used to get the webhooks.
```
